src/components/grid-aware/OneParagraphBlock/OneParagraphBlock.tsx

```diff
@@ -1,11 +1,22 @@
-import PropTypes from "prop-types";
-import React from "react";
+import * as React from "react";
 
-import Button from "../../inline/Button";
+import Button, { ButtonProps } from "../../inline/Button";
 
 import s from "./OneParagraphBlock.module.css";
 
-const OneParagraphBlock = ({ title, descriptions, ctaButtons }) => {
+/* Subcomponents */
```

Comment:
This comment doesn't look right; there's only one component in this file, so if we want to have a comment to demarcate this part of the file, it really ought to say something more like "Main component". However, since there's nothing else in this file but the one and only component, maybe we should just remove this comment, which didn't exist in the original non-TypeScript version?

Reply:
Removed.